articles/changes.md

```diff
@@ -0,0 +1,245 @@
+---
+layout: default
+title: Changes between ROS 1 and ROS 2
+permalink: articles/changes.html
+abstract:
+  This article provides an overview about the changes being made in ROS 2 compared to ROS 1.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Preface
+
+Each change is described as briefly as possible but giving enough context and rational for a reader familiar with ROS 1.
+
+
+## Platforms and dependencies
+
+### Platforms
+
+ROS 1 is only being CI tested on Ubuntu.
+It is actively supported by the community on other Linux flavors as well as OS X.
+
+ROS 2 will be CI tested and supported on Ubuntu, OS X as well as Windows.
+
+
+### Languages
+
+#### C++ standard
+
+ROS 1 is targeting C++03.
+ROS 2 will use C++11.
```

Comment:
I don't think this is the case for VS 2015: http://blogs.msdn.com/b/vcblog/archive/2015/06/19/c-11-14-17-features-in-vs-2015-rtm.aspx

We should probable mention here: `C++11 (and newer) when supported on all major platforms`.